dev-env

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -eu
+
+declare -a POSSIBLE_DOCKER_SOCK_PATHS=(
+    /run/docker.sock
+    /var/run/docker.sock
```

Comment:
These are all locations of where Docker socket file should be present on Linux distributions, to my knowledge.